src/VisualStudio/TestUtilities/IntegrationLog.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+
+namespace Roslyn.VisualStudio.Test.Utilities
+{
+    public sealed class IntegrationLog : IDisposable
+    {
+        private static IntegrationLog _current;
```

Comment:
Let's avoid statics in test helpers...it's a practice that only bites you in the end.